documentation/repl_quickstart.rst

```diff
@@ -20,12 +20,12 @@ In the example below we first define a very basic class called ``Adder``.
 Then we use PCS to track the execution of a method on this class and share the
```

Comment:
Looks like maybe there's a kruft file in this dir?  `documentation/demos/temp.py`

Reply:
Good catch. Removed.